.gitignore

```diff
@@ -75,3 +73,27 @@ distribute-*.tar.gz
 
 # Mac OSX
 .DS_Store
+
+
+# Mercurial
+.hg/
+
+
+# SVN
+.svn/
+
+
+# mypy
+.mypy_cache/
+
+
+# Unit test / coverage reports
+htmlcov
+.coverage
+.pytest_cache/
+.tox/
+.nox/
+
+
+# Environments
+.venv
```

Comment:
Why are there additional entries added to .gitignore?

Reply:
All entries added to .gitignore are from existing Black [configuration](https://github.com/tardis-sn/tardis/blob/351cc8f7b546f9eddf94608dbba2908ed333281b/pyproject.toml#L4). But these folders are just generic cache/build/temporary things, it's a good idea to just ignore them universally. To back it up, all of them are in various .gitignore templates from [gitignore repo](https://github.com/github/gitignore) and there are no such folders in tardis repo so we won't ignore something we shouldn't.